changelogs/fragments/9087-mattermost-priority.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - mattermost - adds support for message priority (https://github.com/ansible-collections/community.general/issues/9068)
```

Minor nit, but needed for consistency.
```suggestion
  - mattermost - adds support for message priority (https://github.com/ansible-collections/community.general/issues/9068).
```

---

Yes I absolutely agree, thanks, that was just my oversight looking at the fragment docs.